STATUS: INLINE SUCCESSFUL SETUPTEARDOWN
MODIFIED: io.jboot.test.cache.j2cache.J2CacheTester.testPut
MODIFIED FILE: /home/lichengpeng/test2/automated_script/yangfuhai/jboot/src/test/java/io/jboot/test/cache/j2cache/J2CacheTester.java
CLEANER: io.jboot.test.cache.caffeine.CaffeineTester.testDue
CLEANER FILE: /home/lichengpeng/test2/automated_script/yangfuhai/jboot/src/test/java/io/jboot/test/cache/caffeine/CaffeineTester.java
POLLUTER: N/A
POLLUTER FILE: N/A
ORIGINAL CLEANER SIZE: 7
NEW CLEANER SIZE: 1
ELAPSED TIME: 36.36500000953674
==========================
@@ -14,0 +14,3 @@
+ {
+     JbootApplication.setBootArg("jboot.cache.type", "caffeine");
+ }
